packages/sui-react-head/src/index.tsx

```diff
@@ -23,6 +25,21 @@ interface HeadProps {
   }>
 }
 
+interface MetaProps extends React.ComponentProps<typeof MetaPrimitive> {}
+
+interface MetaTagInverterProps extends React.MetaHTMLAttributes<HTMLMetaElement> {
+  'data-rh': string
+}
+
+const MetaTagInverter: React.FC<MetaTagInverterProps> = ({ 'data-rh': rh, ...others }) => {
+  return <meta {...others} data-rh={rh} />
+}
+
+const Meta: React.FC<MetaProps> = (props) => {
+  // @ts-expect-error
+  return <MetaPrimitive {...props} tag={MetaTagInverter} />
```

Comment:
we have to add the // @ts in order to solve typescript typing errors related to a "bad" implementation of the types definition in  react-head index.d.ts